tools/ios/build_xcframework.py

```diff
@@ -213,19 +213,64 @@ def platform_arch_framework_build_dir(platform, arch):
             platform_fat_framework_dir.mkdir()
 
             # copy over files from arch-specific framework to fat framework
-            for framework_file_relative_path in [Path("Headers"), Path("Info.plist")]:
-                src = first_arch_framework_dir / framework_file_relative_path
-                dst = platform_fat_framework_dir / framework_file_relative_path
-                if src.is_dir():
-                    shutil.copytree(src, dst)
-                else:
-                    shutil.copy(src, dst)
-
-            # combine arch-specific framework libraries
-            arch_libs = [str(framework_dir / "onnxruntime_extensions") for framework_dir in arch_framework_dirs]
-            run(
-                *([_lipo, "-create", "-output", str(platform_fat_framework_dir / "onnxruntime_extensions")] + arch_libs)
-            )
+            # macos requires different framework structure:
```

Comment:
do we also need this for maccatalyst?

Reply:
According to this [article](https://developer.apple.com/documentation/bundleresources/placing_content_in_a_bundle?language=objc), it seems that we also need the same structure for maccatalyst